Author: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Description: Use unpatched a52
 We don't have liba52 patched to support the A52_USE_DPLII macro,
 so we ignore it for the moment.

Forwarded: no
Last-Update: 2012-06-15
Signed-off-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
--- a/libhb/deca52.c
+++ b/libhb/deca52.c
@@ -109,7 +109,7 @@
             break;
 
         case HB_AMIXDOWN_DOLBYPLII:
-            pv->flags_out = ( A52_DOLBY | A52_USE_DPLII );
+            pv->flags_out = A52_DOLBY;
             break;
 
         case HB_AMIXDOWN_DOLBY:
